gdc_client/defaults.py

```diff
@@ -33,4 +33,8 @@
 # proxy (on `proxy_host`) to traffic on the GDC api host
 proxy_port = 9000
 
-part_size = 1073741824 # 1GiB
+
+MB = 1024 * 1024
```

Comment:
These are unused variables. This is probably fine how it is, otherwise you can just say `1024 * 1024 * 1024` or `1024**3`

Reply:
alright